packages/twenty-server/src/engine/core-modules/auth/enum/impersonation-type.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum ImpersonationTokenTypeEnum {
+  WORKSPACE = 'WORKSPACE',
```

Comment:
why do we need to distinguish both here? I feel it's more a permission issue but does not have to be part of the impersonation token?